pgcontents/query.py

```diff
@@ -407,6 +408,9 @@ def rename_directory(db, user_id, old_api_path, new_api_path):
     old_db_path = from_api_dirname(old_api_path)
     new_db_path = from_api_dirname(new_api_path)
 
+    if old_db_path == '/':
+        raise RenameError('Renaming the root directory is not permitted.')
```

Comment:
We have other kinds of renaming errors already (e.g. `DirectoryExists`).  This should probably just be `RenameRoot`.